demos/README.md

```diff
@@ -1,75 +1,219 @@
+- [File manifest](#file-manifest)
+  - [Important files and folder structure](#important-files-and-folder-structure)
+  - [Supported editors](#supported-editors)
+  - [Technologies with a demo](#technologies-with-a-demo)
+- [Known bugs](#known-bugs)
+- [Change log](#change-log)
+- [News](#news)
+- [Documentation](#documentation)
+- [License](#license)
+- [Contact information](#contact-information)
+
+## Requirements
+
+This project requires basic knowledge of the `git` and `npm` commands.
+It also uses the `npx` tool which comes bundled with `npm`.
+
+To install node and npm follow this [guide-npm](https://docs.npmjs.com/downloading-and-installing-node-js-and-npm), and use this other one [guide-git](https://www.atlassian.com/git/tutorials/install-git) to install git.
+
+
+[Lerna](https://lerna.js.org/) is used as a cornerstone of the project structure. Its main features, and how to install it, are explained in this guide.
 
 ## Instructions
 
-First, follow the instructions of the README file you can find in the root directory of the project.
+Please, follow the instructions to have the best experience when working with this project.
 
-### Adding lerna dependencies
+### Cloning
 
-To work with lerna, you must replace the packages content, with the following lines, in the file lerna.json you can find in the root directory of the project. Or temporarily replace all the content with the one from lerna.demos.json file.
+First, clone the repository with this url: https://github.com/wiris/html-integrations
 
-```javascript
-  "packages": [
-    "demos/**/*",
-    "packages/*"
-  ],
+#### Linux/Mac
+
+```sh
+$ git clone https://github.com/wiris/html-integrations
+$ cd html-integrations
+html-integrations$
+```
+
+#### Windows
+
+You will need to have administrator privileges or activate
+[developer mode](https://docs.microsoft.com/en-us/windows/uwp/get-started/enable-your-device-for-development)
+in your account, and then use the `core.symlinks` when cloning the repository:
+
+```sh
+projects$ git clone --config core.symlinks=true https://github.com/wiris/html-integrations
 ```
 
 ### Quick start
 
-Go on the desired editor folder and run;
+#### Bootstraping
+
+The mono-repository is managed through [Lerna](https://lerna.js.org/), a tool
+designed for maintaining multiple npm packages in a single git repository.
+
+To work with lerna and try out the packages, with your local changes instead of the published packages, you'd need to copy the contents from `lerna.demos.json` to `lerna.json`.
+
+#### Installation
+
+After cloning this repo, and optionally adding lerna dependencies, open a Terminal window to run these next commands:
+
+```sh
+html-integrations$ npm install
+html-integrations$ npm start
+```
+
+Then go on the demos folder, select the technology and the editor and run;
 
 ```
-$ npm i
+html-integrations/demos/[technology]/[editor]$ npm i
```

Comment:
This is not the actual structure right now, correct?

Reply:
No, but the following task involves changing the current structure to the one documented in the README, so the idea of putting it that way is to avoid forgetting in the future or having to make changes right after having made others.